### PR TITLE
provider/kubernetes: HA Deploy first steps

### PR DESCRIPTION
**Pull request by lwander**
There are a few more work items, but this is arguably
the largest chunk. Next I need to
1. Provide configuration for GAFW authentication
2. Provide pipelines for updating & draining components
3. Flush out documentation of individual components

@duftler @ttomsu PTAL

### CODE REVIEW

File: experimental/kubernetes/ha/clouddriver/accounts/clouddriver.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+kubernetes:
```
Comment (duftler) [inline experimental/kubernetes/ha/clouddriver/accounts/clouddriver.yaml]: Is there a reason why some of the files are `.yaml` and others are `.yml`?

File: experimental/kubernetes/ha/clouddriver/cache/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Caching Clouddriver
+
+This set of clouddriver instances exists to periodically scan the configured
+cloud providers infrastructure, and store it in the cache. It is not reachable
+by any other services (hence the empty `/svcs` directory), but the results it
```
Comment (duftler) [inline experimental/kubernetes/ha/clouddriver/cache/README.md]: hence

File: experimental/kubernetes/ha/config/spinnaker-local.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    storage_bucket: spinnaker
+    bucket_root: front50
+
+    cassandra:
+      enabled: false
+    redis:
+      enabled: false
+    gcs:
+      enabled: true
+      project: nits-1132
```
Comment (duftler) [inline experimental/kubernetes/ha/config/spinnaker-local.yaml]: What should the checked-in value be here?

**Comment by duftler** [comment]
LGTM.

File: experimental/kubernetes/ha/clouddriver/accounts/clouddriver.yaml (hunk 1 above)
Comment (lwander) [inline experimental/kubernetes/ha/clouddriver/accounts/clouddriver.yaml]: They should all be .yaml, I'll find the stragglers

File: experimental/kubernetes/ha/clouddriver/cache/README.md (hunk 2 above)
Comment (lwander) [inline experimental/kubernetes/ha/clouddriver/cache/README.md]: tangibly

File: experimental/kubernetes/ha/config/spinnaker-local.yaml (hunk 3 above)
Comment (lwander) [inline experimental/kubernetes/ha/config/spinnaker-local.yaml]: Not that, good catch

File: experimental/kubernetes/ha/config/spinnaker.yaml [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+    # You need to provide the fully-qualified path to the directory containing the packer templates.
+    # They typically live in rosco's config/packer directory.
+    configDir: /opt/rosco/config/packer
+
+  bakery:
+    host: ${services.rosco.host}
+    port: ${services.rosco.port}
+    baseUrl: ${services.rosco.baseUrl}
+    extractBuildDetails: true
+    propagateCloudProviderType: true
```
Comment (duftler) [inline experimental/kubernetes/ha/config/spinnaker.yaml]: All `propagateCloudProviderType` references can be dropped.

File: experimental/kubernetes/ha/config/spinnaker.yaml [hunk 5]
```diff
@@ -0,0 +1,315 @@
+# This file is intended to serve as a master configuration for a Spinnaker
+# deployment. Customizations to the deployment should be made in another file
+# named "spinnaker-local.yml". The distribution has a prototype called
```
Comment (duftler) [inline experimental/kubernetes/ha/config/spinnaker.yaml]: Should the filenames in these comments also end with `.yaml`?

Comment (duftler) [inline experimental/kubernetes/ha/config/spinnaker.yaml]: Quite a few `.yml` references in the comments in this file.

File: experimental/kubernetes/simple/config/clouddriver.yml [hunk 6]
```diff
@@ -0,0 +1,82 @@
+server:
```
Comment (duftler) [inline experimental/kubernetes/simple/config/clouddriver.yml]: Are we sticking with `.yml` files in the 'simple' setup?

**Pull request merged**